src/Microsoft.XmlSerializer.Generator/pkg/build/Microsoft.XmlSerializer.Generator.props

```diff
@@ -0,0 +1,5 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <DotNetCliToolReference Include="Microsoft.XmlSerializer.Generator" Version="1.0.0-preview2-25628-0" />
```

Comment:
We need to set the version according to the actual package version. Do we have issue for tracking that? If not, can you please open an issue for tracking that?

Reply:
#23691